learn-data-structures-by-building-a-shopping-cart/index05.js

```diff
@@ -0,0 +1,35 @@
+const products = [
+  {
+    id: 0,
+    name: "Vanilla buttercream cupcake",
+    price: 2.99
+  },
+  {
+    id: 1,
+    name: "French Macaroon",
+    price: 3.99
+  },
+  {
+    id: 2,
+    name: "Fruit sprinkles cupcake",
+    price: 3.99
+  },
+  {
+    id: 3,
+    name: "Pink flower cupcake",
+    price: 5.99
+  }
+];
+
+class ShoppingCart {
+  constructor() {
+
+  }
+}
+
+/*
+When we declare values inside our constructor, we attach the 'this' keyword in front of them.
```

Comment:
When we assign properties inside our constructor, we attach the 'this' keyword in front of them.
Then each new object created by the class will have those properties.
Create an empty array and assign it to a variable called 'items' inside the constructor.
For example, `this.make = 'Hyundai';`